plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -48,6 +48,13 @@ func pathVenafiCertEnroll(b *backend) *framework.Path {
 				Type:        framework.TypeCommaStringSlice,
 				Description: "Use to specify custom fields in format 'key=value'. Use comma to separate multiple values: 'key1=value1,key2=value2'",
 			},
+			"ttl": {
+				Type: framework.TypeDurationSecond,
+				Description: `The lease duration if no specific lease duration is
+requested. The lease duration controls the expiration
+of certificates issued by this backend. Defaults to
+the value of max_ttl.`,
+			},
```

Comment:
This description applies to roles not certificates.  Please change to the following (which is basically the same as the native Vault PKI's path-help description for the `ttl` parameter):
> The requested Time To Live for the certificate; sets the expiration date. If not specified the role default is used. Cannot be larger than the role max TTL.

Reply:
Fixed on the latest commit.